### DISCUSSION TITLE
Error: 'Id is required' (for existing database)

### DISCUSSION THREAD

**Discussion by arladmin** [directus/directus#17540] [Archive]
### Describe the Bug

I've installed the latest Directus on an existing Postgres database, where the tables already have an `id` field of type `int4` with `Primary Key` constraints defined on them.

These fields show correctly in Directus:
![image](https://user-images.githubusercontent.com/64059154/217813720-162546f4-2c6a-4b8c-8c77-fdcb75df0a46.png)

but when i try to add an item, i see the error:
![image](https://user-images.githubusercontent.com/64059154/217813848-beabf90a-23f2-4b2d-8d78-c955c5c7dba8.png)

**Comment by rijkvanzanten** [comment]
Does your column have a default value? If not, you'll have to submit an ID from Directus when creating an item. In this case, the field is hidden. Make sure to make the field visible from the field options so the user can enter an ID
